components/more_channels/index.js

```diff
@@ -24,6 +24,7 @@ function mapStateToProps(state) {
         channels: getNotArchivedOtherChannels(state) || [],
         teamId: team.id,
         teamName: team.name,
+        channelsRequestStatus: state.requests.channels.getChannels.status === RequestStatus.STARTED,
```

Comment:
Based on the name `channelsRequestStatus` I would expect a status string rather than a boolean. The name `channelsRequestStarted` is more boolean-like in this case.